news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,133 @@
+### New features & improvements
+
+- The orders pages have been migrated to symfony, redesigned and new features have been added to avoid back and forth between orders and improve efficiency.
+For more details, read [the article]() about all the new features of the order pages.
+
+-  Fuzzy search: Improve search mechanism, allow mistakes in search word. Developed by a community member, the great [Lathanao](https://github.com/Lathanao), with the help of the core team, the fuzzy search feature implements the Levenshtein algorithm to improve the PrestaShop’s native search functionality by taking into account misspelling or error inputs. Read more [here](https://build.prestashop.com/news/introduction-to-the-fuzzy-search) ! 
+
+- Currency management: Add new official and non official currencies to your store and customize their display per language as desired 
+
+![Customize currencies symbol and format](/assets/images/2020/04/Currencies-customize_symbol_and_format.png)
+
+![Currencies format by language](/assets/images/2020/04/Currencies-format_by_language.png)
+
+- The field Manufacturer Part Number is now available for product management. It's used to identify a specific product of a given manufacturer. It will improve indexing and searching, and also have a positive effect on SEO.
+
+- An option has been added In Shop parameters > General to choose to enable or disable suppliers in front-office, regardless of brands. There used to be only one generic option, which could be problematic for merchants. They are now separated, to suit all merchants’ needs.
+
+- Error page in the Back office: If an unexpected error occurs in the back office when running in production mode, you are now presented with a useful error page instead of a blank page, this way you have more information to understand what is happening.
+
+- It is now possible to translate email contents in the Back Office, when using the new Email theme feature introduced in 1.7.6 and available in Design > Email theme 
+
+- A lot of [international improvements](https://build.prestashop.com/news/prestashop-keeps-growing-internationally/) have been made. New languages have been added to PrestaShop and its installer, localization packs have been updated to keep PrestaShop up-to-date to offer accurate international data for all businesses.
+
+
+### Improved core modules
+
+Some modules have been improved with bug fixes for more robustness and new features for more productivity :
+
+- Theme customization: Now available for product & category pages (in addition to the home page), you can easily configure and customize your product and categories’ pages theme and main native modules.
+
+- Block reassurance: Now available on all pages (above or below the header and / or the footer) in addition to product pages and during the checkout. You can customize the color of the icon, add a description and a link. With this module you build the confidence of your visitors and thus improve your conversion rate.
```

Comment:
check module name in english

Reply:
The technical name is "block reassurance" and in the BO the name displayed is "Customer Reassurance"

Reply:
```suggestion
- Customer reassurance (blockreassurance): Now available on all pages (above or below the header and / or the footer) in addition to product pages and during the checkout. You can customize the color of the icons, add a description and a link. With this module you build the confidence of your visitors and thus improve your conversion rate.
```